assignments/prototype-refactor.js

```diff
@@ -7,3 +7,164 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
```

Comment:
Awesome use of your initial commit. This is a significant change of the file and you've handled it perfectly.